benchmarks/startup/config.yaml

```diff
@@ -0,0 +1,32 @@
+startup_with_ddtrace_run: &defaults
+  use_ddtrace_run: "true"
+  telemetry_enabled: "false"
+  runtime_metrics_enabled: "false"
+  with_trace: "false"
+
+startup_without_ddtrace_run:
+  <<: *defaults
+  use_ddtrace_run: "false"
+
+startup_with_telemetry:
+  <<: *defaults
+  use_ddtrace_run: "false"
+  telemetry_enabled: "true"
+
+ddtrace_run_with_telemetry:
+  <<: *defaults
+  telemetry_enabled: "true"
+
+ddtrace_run_with_runtime_metrics:
+  <<: *defaults
+  runtime_metrics_enabled: "true"
+
+ddtrace_run_then_send_trace:
```

Comment:
We do not need to cover sending traces in this startup benchmark.

Reply:
I renamed this benchmark so it's clear that we are mocking all outbound requests using httpretty. The key here is to evaluate the performance of encoding traces and telemetry payloads. I could mock `AgentWriter._put` and `TelemetryWriter._send_payload` instead of using httpretty but I think the current approach provides the most accurate test